operator/pkg/reconciliation/reconcile_reaper.go

```diff
@@ -15,52 +15,60 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
 
-	api "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
 	"github.com/datastax/cass-operator/operator/internal/result"
+	api "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
 	"github.com/datastax/cass-operator/operator/pkg/images"
 )
 
 const (
-	ReaperUIPort             = 7080
-	ReaperAdminPort          = 7081
-	ReaperDefaultPullPolicy  = corev1.PullIfNotPresent
-	ReaperContainerName      = "reaper"
-	ReaperHealthCheckPath    = "/healthcheck"
-	ReaperKeyspace           = "reaper_db"
-	ReaperSchemaInitJob      = "ReaperSchemaInitJob"
+	ReaperUIPort            = 7080
+	ReaperAdminPort         = 7081
+	ReaperDefaultPullPolicy = corev1.PullIfNotPresent
+	ReaperContainerName     = "reaper"
+	ReaperHealthCheckPath   = "/healthcheck"
+	ReaperKeyspace          = "reaper_db"
+	ReaperSchemaInitJob     = "ReaperSchemaInitJob"
 	// This code currently lives at https://github.com/jsanda/create_keyspace.
 	ReaperSchemaInitJobImage = "jsanda/reaper-init-keyspace:latest"
 )
 
-func buildReaperContainer(dc *api.CassandraDatacenter) corev1.Container {
+func buildReaperContainer(dc *api.CassandraDatacenter, container *corev1.Container) {
```

Comment:
let's add a docstring to explain why we're modifying the input instead of returning something

Reply:
also let's add a note about adding a unit test here - but I know @jsanda has plans to change this code so no need to do it now